location_tag.md

```diff
@@ -0,0 +1,110 @@
+# Location Tag Type
+
+## Overview
+
+* Tags are strings attached to an entity (like tagging of blog entries) to form a folksonomy and/or to categorize data. 
+
+* Location Tags are being used to tag locations.Which enables the system to act based on the presence of tags on certain locations. 
```

Comment:
For example, location tags may be used to control which locations are included in a choice list or to define which locations are included in a report.

Place space between period and beginning of next sentence.